apps/admin-ui/components/Layout/Header/Breadcrumbs.tsx

```diff
@@ -0,0 +1,60 @@
+import { Box, Breadcrumbs, Link, Typography } from "@mui/material";
+import { BreadcrumbsNameMaps } from "./Header";
+import { useRouter } from "next/router";
+
+interface BreadcrumbProps {
+    breadcrumbs: BreadcrumbsNameMaps
+}
```

Comment:
no need to create this `props` interface with only one value in it.
send that value directly.